server/src/com/cloud/server/ConfigurationServerImpl.java

```diff
@@ -894,6 +894,13 @@ protected void injectSshKeysIntoSystemVmIsoPatch(String publicKeyPath, String pr
         } else {
             command = new Script("/bin/bash", s_logger);
         }
+        String osname = System.getProperty("os.name");
```

Comment:
Hi  karuturi ,

thanks for your kindly help , yes , I should use the isOnWindows function .

ChunFeng